cts/cli/regression.tools.exp

```diff
@@ -23,6 +23,585 @@ A new shadow instance was created. To begin using it, enter the following into y
 </cib>
 =#=#=#= End test: Validate CIB - OK (0) =#=#=#=
 * Passed: cibadmin       - Validate CIB
+=#=#=#= Begin test: List all available cluster options =#=#=#=
+cluster-options - Pacemaker cluster options
+
+Also known as properties, these are options that affect behavior across the entire cluster. They are configured within cluster_property_set elements inside the crm_config subsection of the CIB configuration section.
+
+dc-version
```

Comment:
This format is very much open to feedback

Reply:
I like getting rid of the short/long description headers -- the user doesn't care, and no one needs to parse it.

Putting the short description on the same line as the option name is pretty reasonable. It might look "off" for options that don't have a short desc... maybe in those cases put the long desc after the colon and omit the long desc line.

This is closer to the format of a fancy list. The main problem was that the way fancy lists are implemented, formatting it correctly for fancy would format it incorrectly for non-fancy. However, now I'm thinking we could just enable fancy (via a new internal function) unconditionally. This is new, so no one's relying on old text output. If I can get it working, that's probably the way to go.

Allowed values all on one line is probably better than separate lines. Again, no one is parsing this output.

I'm hesitant to overload "Allowed values" to list either a type or values, but it's probably fine. Listing a value type is meaningless for enumerated values (it's trivially a string type). And we can be less strict with the semantics of each field in text output compared to XML output.

Overall, yeah I think we should aim for something like this. I appreciate the idea. And I don't mind if this PR sits around for a little bit while we tweak the output, or if we revisit it later.

Reply:
Hmm what about translations? We could put the translation of short desc in parentheses at the end of the header line, and the translation of long desc after the English long desc on the indented line... or skip English altogether if a translation exists, though I doubt that's a good idea.

Reply:
Similarly, for the top-level short and long desc, we could put the translation in parentheses in the same paragraph or below it in a new paragraph...

Reply:
The point of translation is to use it instead of English. The layout would remain the same, it would just use the translation.

The only exception is OCF metadata XML, since OCF itself supports translations -- in that case, we always show the English, and add the locale's translation if available. You could make an argument that the XML output here should do the same, but the text output should show only the translation.

Reply:
That makes sense. I was going by the fact that we have historically included both languages (when output was always XML). What you're saying is easier and cleaner.